bot/exts/info/help.py

```diff
@@ -174,7 +174,7 @@ async def command_formatting(self, command: Command) -> Embed:
             command_details += f"**Can also use:** {aliases}\n\n"
 
         # check if the user is allowed to run this command
-        if not await command.can_run(self.context):
+        if not command.enabled or not await command.can_run(self.context):
```

Comment:
Disabled commands should get a separate message which is more specific e.g. "This command is disabled"